app/views/includes/contact.jade

```diff
@@ -0,0 +1,22 @@
+//- CONTACT ::
+section#contact
+    .contact-title.left
+        h4.titulo #{contact.contact_title}
+        span.bold.white
+            p.ct-t1 #{contact.contact_subtitle}
+            p.ct-t2 #{contact.call_to_action}
+    .contact-form
+        form#contactForm.ct-form(action='mailto:info@upplication.com')
```

Comment:
the "mailto:info@upplication.com" is ok, but the body of the email must contain the text of the textarea.
I think we must generate the link using javascript and append to the action to generate a final action like this:
"mailto:info@upplication.com?subject=subject-taken-from-the-input-by-js&body=body-taken-from-the-textarea-by-js&from=from-taken-from-the-input-by-js"